src/build/wrapped-babel-env.js

```diff
@@ -6,7 +6,7 @@ module.exports = function(...args) {
   const dirname = args[2]; 
   
   options.targets = {
-    "node": "10.9.0"
+    "node": "8"
```

Comment:
this should have been 8 this whole time.  I think this is why I was having problems pushing to amazon lambda (they only support 6 and 8).  I gave up after 3 min or errors, I'll retry soon for funz.